hashes.lst

```diff
@@ -25,6 +25,8 @@
 # DES) must be last.
 #
 #name    gs_suffix  h_prefix  nrbytes  flags
+yescrypt :          $y$       15       STRONG
+yescrypt :          $7$       15       STRONG
```

Comment:
Why 15? Shouldn't these be 16? Somehow they're 15 for SHA-crypt as well. I guess it's to more optimally use the base-64 chars, arriving at a whole number of them. I don't really mind, but is it common and would everyone be comfortable with it? I imagine some compliance guidelines could require 128-bit or larger salts, and then it'd be a pity that a system uses 120-bit ones instead.

Reply:
As discussed, these 15's should be changed to 16's.